app/models/bid_status_presenter_factory.rb

```diff
@@ -100,23 +100,23 @@ def ineligible?
   end
 
   def work_in_progress?
```

Comment:
Yes, it was a bit of a bd match before. I wasn't sure if it should go in the Auction model or not since we have been trting to minimize what is in there. Thoughts?